testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -5013,6 +5013,43 @@ public void testUpdateMultipleCondition()
         }
     }
 
+    @Test
+    public void testUpdateWithNullValues()
```

Comment:
Please check the existing `testUpdateNotNullColumn` test. Some assertion look redundant.